### PR TITLE
add new property called `class`

### PR DESCRIPTION
**Pull request by abr4xas**
Add new property called `class` to be able to add CSS styles

### CODE REVIEW

**Comment by abr4xas** [comment]
Hi there @gehrj please, check this!! :+1:

File: src/components/country-select.vue [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+      }
     }
-  }
-})
+  })
 </script>
 
 <template>
-  <select @change="onChange($event.target.value)">
+  <select @change="onChange($event.target.value)" class="class">
```
Comment (gehrj) [inline src/components/country-select.vue]: please change class to :class="className". You need the : to achieve what you are going for here.

File: README.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ topCountry | no | '' | string | By providing this value you will tell component
 countryName | no | false | boolean | By setting this value to true, country names will be output in full instead of using the abbreviated short codes. Make sure to set this true for both country and region if you are using.
 whiteList | no | [] | array | Fill this array with capitalized short codes of the countries you want to appear in the dropdown list. ex: ['US', 'CA', 'MX']
 blackList | no | [] | array | Fill this array with capitalized short codes of the countries you want to remove from dropdown list. ex: ['US'] 
+class | no | [] | string | Class name ex: `form-control`
```
Comment (gehrj) [inline README.md]: change class to className as class is a javascript keyword. Also change [] to '' as default value is string not array.

File: README.md [hunk 3]
```diff
@@ -60,6 +61,7 @@ country | no | '' | string | This tells the component what country to grab the l
 defaultRegion | no | 'US' | string | This allows you to set a default region when choosing not to use the country attribute. It will be set to regions of the United States if not provided.
 countryName | no | false | boolean | Set this to true if you are setting it to true while using Country Select. This is just to help keep the data values in sync.
 regionName | no | false | boolean | Set this to true if you want the v-model to output full region names instead of the default abbreviations.
+class | no | [] | string | Class name ex: `form-control`
```
Comment (gehrj) [inline README.md]: change class to className as class is a javascript keyword. Also change [] to '' as default value is string not array.

File: src/components/country-select.vue [hunk 4]
```diff
@@ -1,55 +1,55 @@
...
-        countryList = countryList.filter((country) => {
-          return this.whiteList.includes(country.countryShortCode)
-        })
-      }
-      if (this.blackList) {
-        countryList = countryList.filter((country) => {
-          return !this.blackList.includes(country.countryShortCode)
+  export default Vue.extend({
+    name: 'CountrySelect',
+    props: ['country', 'topCountry', 'countryName', 'whiteList', 'blackList', 'class'],
```
Comment (gehrj) [inline src/components/country-select.vue]: change class to className

File: src/components/region-select.vue [hunk 5]
```diff
@@ -1,70 +1,70 @@
...
-    if (this.country) {
-      this.getRegionWithCountry()
-    } else {
-      let findRegion = ''
-      if (this.countryName) {
-        findRegion = this.defaultRegion ? this.defaultRegion : 'United States'
+  import regions from 'country-region-data'
+  export default {
+    name: 'RegionSelect',
+    props: ['country', 'region', 'defaultRegion', 'countryName', 'regionName', 'class'],
```
Comment (gehrj) [inline src/components/region-select.vue]: change class to className

File: src/components/region-select.vue [hunk 6]
```diff
@@ -1,70 +1,70 @@
...
-        this.shownRegions = []
       }
     }
   }
-}
 </script>
 
 <template>
-  <select @change="onChange($event.target.value)">
+  <select @change="onChange($event.target.value)" class="class">
```
Comment (gehrj) [inline src/components/region-select.vue]: please change class to :class="className". You need the : to achieve what you are going for here.

**Comment by gehrj** [comment]
@abr4xas Please take a look at the comments and make suggested changes and then I will merge it in. I like the idea!

**Comment by abr4xas** [comment]
Hey man, sure... :+1:

**Pull request merged**